website/docs/r/system_center_virtual_machine_manager_server.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "System Center Virtual Machine Manager"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_system_center_virtual_machine_manager_server"
+description: |-
+  Manages a System Center Virtual Machine Manager Server.
+---
+
+# azurerm_system_center_virtual_machine_manager_server
+
+Manages a System Center Virtual Machine Manager Server.
+
+~> **Note:** By request of the service team the provider no longer automatically registering the `Microsoft.ScVmm` Resource Provider for this resource. To register it you can run `az provider register --namespace Microsoft.ScVmm`.
+
+-> **Note:** This resource depends on an existing `System Center Virtual Machine Manager Host Machine`, `Arc Resource Bridge` and `Custom Location`. Installing and configuring these dependencies is outside the scope of this document. See [Virtual Machine Manager documentation](https://learn.microsoft.com/en-us/system-center/vmm/?view=sc-vmm-2022) and [Install VMM](https://learn.microsoft.com/en-us/system-center/vmm/install?view=sc-vmm-2022) for more details of `System Center Virtual Machine Manager Host Machine`. See [What is Azure Arc resource bridge](https://learn.microsoft.com/en-us/azure/azure-arc/resource-bridge/overview) and [Overview of Arc-enabled System Center Virtual Machine Manager](https://learn.microsoft.com/en-us/azure/azure-arc/system-center-virtual-machine-manager/overview) for more details of `Arc Resource Bridge/Appliance`. See [Create and manage custom locations on Azure Arc-enabled Kubernetes](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/custom-locations) for more details of `Custom Location`. If you encounter issues while configuring, we'd recommend opening a ticket with Microsoft Support.
```

Comment:
According to this [Architecture figure](https://learn.microsoft.com/en-us/azure/azure-arc/system-center-virtual-machine-manager/overview#architecture), this resource depends on a connected `Arc Resource Bridge` and `SCVMM Management Server`. So the first sentence should not contain `Custom Location` ?

Reply:
It's [required property](https://github.com/Azure/azure-rest-api-specs/blob/d1eb3c20113d1018f25a8d97fdfa5f8bb5c659ea/specification/scvmm/resource-manager/Microsoft.ScVmm/stable/2023-10-07/scvmm.json#L2924) so that we need to call it out in tf doc.